Readme.md

```diff
@@ -313,6 +313,107 @@ cardNumber.mount('#card-number-container').then(function() {
 ...
 ```
 
+
+#### Field Creation Options
+`.create(field, options)` method accepts an optional second argument `options` which is helpful in case you want customize the field created with it.
```

Comment:
```suggestion
`.create(field, options)` method accepts an optional second argument `options` which is helpful in case you want to customize the field created with it.
```